huaweicloud/services/vpcep/resource_huaweicloud_vpcep_endpoint.go

```diff
@@ -57,14 +57,15 @@ func ResourceVPCEndpoint() *schema.Resource {
 			},
 			"network_id": {
 				Type:     schema.TypeString,
-				Required: true,
+				Optional: true,
+				Computed: true,
 				ForceNew: true,
 			},
 			"enable_dns": {
 				Type:     schema.TypeBool,
 				Optional: true,
+				Computed: true,
 				ForceNew: true,
-				Default:  true,
```

Comment:
Remove `Computed`, and add `Default:  true`

Reply:
done